### DISCUSSION TITLE
Have you noticed the extra pads on the Pro Mini style LQFP32 ?

### DISCUSSION THREAD

**Discussion by SuperUserNameMan** [dbuezas/lgt8fx#239] [Ideas]
Hello.

Have you noticed the 2 little SMD pads next to the red power LED of the Pro Mini style LQFP32 ?

The pad close to the LED is GND. 
The other pad is connected to the pin 07 of the chip (PB6 / XTALO ).

I thought it could be used to add an external oscillator, but PB7 / XTALI is on pin 08 which is not connected to anything. 

Then I thought it could be used to solder an extra LED, but there is no room for a resistor.

Any idea what could be the purpose of this extra pad ?

PS : my version is 3.3V and the version of the picture below is 5V, so it seems not to be a way to tell to the program which version of the regulator. Though, if this pin is used as a digital input, it could be used as a solder switch to activate a debug mode or change a configuration parameter or whatever ... 

![](https://raw.githubusercontent.com/dbuezas/lgt8fx/master/docs/boards/pro_mini_LQFP32.png)

-------

LGT8FX8P_databook v1.0.5-English.pdf : page 6
> PB6 / XTALO :
> - PB6 : programmable interface B6.
> - XTALO : oscillator IO output interface.

LGT8FX8P_databook v1.0.5-English.pdf : page 84
> Port B Pins Alternate Functions
> PB6 : 
> - XTALO / TOSC1 ( External Master Oscillator pin XO )
> - PCINT6 ( Pin change interrupt 6)

LGT8FX8P_databook v1.0.5-English.pdf : page 85
> XTALO/TOSC1/PCINT6- Port B Pin 6
> - XTALO: external crystal oscillator pin XO
> - TOSC1: Timer external Oscillator pin 1. Used only if internal RC is selected as master working clock, and the asynchronous timer is enabled by the correct setting in ASSR register. When the AS2 bit in ASSR is set (one) and the EXCLK bit is cleared (zero) to enable asynchronous clocking of Timer/Counter2 using the Crystal Oscillator, PB6 is disconnected from internal I/O port, and becomes the input pin of the Oscillator amplifier. In this mode, external crystal Oscillator is connected to this pin.
> - PCINT6: Pin Change Interrupt 6. The PB6 pin can serve as an external interrupt source. If PB6 is used as a crystal oscillator pin, DDB6, PORTB6 and PINB6 will all read 0.

**Comment by SuperUserNameMan** [comment]
I thought it could be a secret switch to make the flash read only, but it does not seem to have any effect on the bootloader.

**Comment by LaZsolt** [comment]
> PS : my version is 3.3V and the version of the picture below is 5V, so it seems not to be a way to tell to the program which version of the regulator.

Here is a way to figure out your board's regulator voltage: https://github.com/dbuezas/lgt8fx/discussions/123#discussion-3287695

**Comment by LaZsolt** [comment]
The secret of the extra pad if they solder the LGT8F328D is:

![kép](https://user-images.githubusercontent.com/60486174/170876533-406057b9-0693-42e4-b4ec-573eff80e058.png)

**Comment by dwillmore** [comment]
@LaZsolt seems this issue can be close and still serve as a reference in the future.

**Comment by SuperUserNameMan** [comment]
oh, forgot about this.
Was not really an issue. I think the forum was not available back then when i posted.
Maybe this could be moved into the forum ?

**Reply by dwillmore** [reply]
Yes!  Done.
